.circleci/config.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         # We do this instead of setting --default-tf-version because setting
```

Comment:
If dependabot and renovatebot will clash (I'm sure they would), then let's configure renovatebot to handle the same dependencies as dependabot, and then let's remove the dependabot file.

https://github.com/runatlantis/atlantis/blob/main/.github/dependabot.yml

Reply:
Currently, dependabot manage `gomod`, `docker`, and `github-actions`.

When introducing renovate, all of them are supported like https://github.com/krrrr38/atlantis/issues/4. so removed dependabot.yml.

<img width="290" alt="image" src="https://user-images.githubusercontent.com/915731/208287910-95e7b94a-beb8-44de-9c81-380fef89779a.png">